armi/reactor/reactors.py

```diff
@@ -435,12 +435,27 @@ def removeAssembly(self, a1, discharge=True):
         else:
             self._removeListFromAuxiliaries(a1)
 
-    def removeAssembliesInRing(self, ringNum):
+    def removeAssembliesInRing(self, ringNum, forceHexRing=False):
         """
         Removes all of the assemblies in a given ring
```

Comment:
add comment similar to the one in `getAssembliesInRing` or a `See Also` clause to that method where the definition of "ring" is given.

Reply:
Done